### PR TITLE
Geospatial and API localization

### PR DESCRIPTION
**Pull request by noirbizarre**
This pull-request:

- [x] internationalize spatial granularities and territories
- [x] ensure Admin API calls are localized
- [x] ensure spatial APIs and completer are localized (ie .World vs "Monde" for french localization)

### CODE REVIEW

File: js/api.js [hunk 1]
```diff
@@ -3,6 +3,30 @@ import SwaggerClient from 'swagger-client';
 import config from 'config';
 import log from 'logger';
 
+
+function updateQueryParam(uri, key, value) {
+    const re = new RegExp(`([?&])${key}=.*?(&|$)`, 'i');
```
Comment (davidbgk) [inline js/api.js]: Add comments for both the function and the regexp ;)

File: udata/core/spatial/api.py [hunk 2]
```diff
@@ -67,7 +68,7 @@ def get(self):
         return [
             {
                 'id': opt['text'],
-                'name': opt['payload']['name'],
+                'name': _(opt['payload']['name']),
```
Comment (davidbgk) [inline udata/core/spatial/api.py]: Wow, does it work dynamically like this?! I'm impressed :+1:

File: udata/core/spatial/translations.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# Here to force translations by gettext
+_ = lambda s: s
+
+TRANSLATIONS = (
```
Comment (davidbgk) [inline udata/core/spatial/translations.py]: Oh OK I get it, so it's not exactly dynamic. How can we maintain that list? Is there a way to check that we have all entries with a command?

**Comment by davidbgk** [comment]
r+wc

File: udata/core/spatial/api.py (hunk 2 above)
Comment (noirbizarre) [inline udata/core/spatial/api.py]: It works only if the string are in the po/mo files.

File: udata/core/spatial/translations.py (hunk 3 above)
Comment (noirbizarre) [inline udata/core/spatial/translations.py]: There was two alternative: the easy one in which we manually maintain a label list in uData.
The other one is to import translations from GeoZones.
I add to manually test the translations (on card) to check that every string is translated.

**Pull request merged**